.gitignore

```diff
@@ -132,6 +132,7 @@ gen/
 .gradle/
 build/
 */build/
+android/app/gradle*
```

Comment:
Ah. Normally the way I do development with is to open the demo app project, instead of the library project itself in Android Studio. The demo app project is configured to include the library project as a dependency so those code files would show up.